pkg/api/controller.go

```diff
@@ -2996,6 +3014,36 @@ func paginationAmount(v *PaginationAmount) int {
 	return i
 }
 
+func resolvePathList(objectList *[]string, prefixList *[]string) []catalog.PathRecord {
+	var pathList []catalog.PathRecord
+	if objectList == nil && prefixList == nil {
+		return make([]catalog.PathRecord, 0)
+	}
+	if objectList != nil {
+		for _, path := range *objectList {
+			if path != "" {
+				path := path
```

Comment:
I'm confused by this line, why do we need it?

Reply:
This is because of linter. I assign path later in line 3027, but it's address, and as I understand it shouldn't take the address of a loop variable.  This is why I reassign path variable inside the loop.